packages/messageport/tizen/src/messageport_manager.cc

```diff
@@ -0,0 +1,244 @@
+// Copyright 2021 Samsung Electronics Co., Ltd. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+#include "messageport_manager.h"
+
+#include <bundle.h>
+#include <flutter/standard_message_codec.h>
+
+#include <cstdint>
+#include <vector>
+
+#include "log.h"
+
+MessagePortManager::MessagePortManager() {}
+
+MessagePortManager::~MessagePortManager() {
+  for (auto& pair : local_ports_) {
+    int port = pair.second;
+    bool is_trusted = pair.first.second;
+    int ret = MESSAGE_PORT_ERROR_NONE;
+    if (is_trusted) {
+      ret = message_port_unregister_trusted_local_port(port);
+    } else {
+      ret = message_port_unregister_local_port(port);
+    }
+
+    if (MESSAGE_PORT_ERROR_NONE != ret) {
```

Comment:
The style is not consistent within the file.
```suggestion
    if (ret != MESSAGE_PORT_ERROR_NONE) {
```